examples/make_titanic_example_data.py

```diff
@@ -10,8 +10,14 @@
 
 import logging
 import os
+import sys
 from itertools import chain
 
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.insert(0, os.path.abspath('../../'))
```

Comment:
This should not be necessary @srhrshr. Did you run `pip install -e .` before running the tests locally?

Reply:
@desilinguist - yep that fixes it! I was doing `pip install -r requirements.txt`. Thanks! Reverting this change.